install_config/router/default_haproxy_router.adoc

```diff
@@ -82,23 +78,20 @@ ifdef::openshift-enterprise[]
 The default router service account, named *router*, is automatically created during quick and advanced installations. To verify that this account already exists:
 endif::[]
 ifdef::openshift-origin[]
-First, ensure you have xref:creating-the-router-service-account[created the
+First, ensure you have xref:index.adoc#creating-the-router-service-account[created the
```

Comment:
xref:../install_config/router/index.adoc#creating-the-router-service-account[created the router service account]